README.md

```diff
@@ -212,12 +217,56 @@ import { AsgardeoSPAClient } from "@asgardeo/auth-spa/polyfilled/umd";
 
 ## APIs
 
+### Using APIs
+
+Asgardeo Auth React SDK is built on top of Asgardeo Auth SPA SDK. Hence, almost all the usable APIs from Auth SPA SDK are re-exported from Asgardeo Auth React SDK.
```

Comment:
Shall we move the best practices to a separate space. WDYT?

Reply:
Fixed with 81c7b4850d4d3cf2c0f5c5b053206079c2ab49e8